glossary.md

```diff
@@ -19,6 +19,7 @@ Nếu một từ chưa có trong bảng thuật ngữ dưới đây, các bạn
 | accuracy                          | độ chính xác                                                   |                                                              |
 | activation function               | hàm kích hoạt                                                  |                                                              |
 | algorithm's performance           | chất lượng của thuật toán                                      |                                                              |
+| artificial data synthesis           | tổng hợp dữ liệu nhân tạo                                      |                                                              |
```

Comment:
Bạn có thể dẫn cả câu có cụm này được không?

Reply:
Đây nha bạn @tiepvupsu 

Chương 43:
Tiêu đề: # 43. Artificial data synthesis
Câu #1: "More generally, there are several circumstances where artificial data synthesis allows you to create a huge dataset that reasonably matches the dev set."
Câu #2: " Keep in mind that artificial data synthesis has its challenges: it is sometimes easier to create synthetic data that appears realistic to a person than it is to create data that appears realistic to a computer."

Reply:
Từ "tổng hợp" nghe có vẻ vẫn chưa rõ nghĩa nhỉ? Cảm giác vẫn hơi bị nhầm sang ý là tập hợp các dữ liệu nhân tạo lại. Nhưng mình cũng chưa nghĩ ra được từ gì hay hơn. 

Từ artificial data đã dịch từ khóa chưa nhỉ? Mình nghĩ nên để từ artificial data làm từ khóa thôi?

Reply:
Uh từ "synthesis" lúc đầu mình cũng phân vân nên là "tổ hợp" hay "tổng hợp". Theo đây (https://www.rung.vn/dict/en_vn/Synthesis ) thì từ này gần với tổng hợp hoá học.
Trong chương 43 thì cụm 3 chữ này đi chung, thực ra thì mình có search toàn project thì không còn chỗ nào sử dụng "Artificial data" hay cả cụm, nên thêm từ này vô chỉ trong trường hợp sau này tác giả có bản update thì mình sài dịch tiếp thôi.  @ngcthuong @tiepvupsu

Reply:
Mình ok với cách dịch này. Sau này có vấn đề gì thì có thể sửa lại.